### PR TITLE
Order Calldata Cli Command 

### PR DESCRIPTION
**Pull request by Siddharth2207**
## Motivation
- Add CLI command to generate addOrder callData for a specified deployment for a particular composition.

## Checks

By submitting this for review, I'm confirming I've done the following:
- [x] made this PR as small as possible
- [x] unit-tested any new functionality
- [x] linked any relevant issues or PRs

### CODE REVIEW

File: crates/cli/src/commands/order/mod.rs [hunk 1]
```diff
@@ -29,6 +32,12 @@ pub enum Order {
 
     #[command(about = "Compose a .rain order file to Rainlang", alias = "comp")]
     Compose(Compose),
+
+    #[command(
+        about = "Generate calldata for addOrder from a composition",
+        alias = "call"
+    )]
+    Calldata(Calldata),
```
[the remark this answers is not in the document]
Comment (Siddharth2207) [reply crates/cli/src/commands/order/mod.rs]: Resolved

File: crates/cli/src/commands/order/orderbook.rs [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+use crate::output::{output, SupportedOutputEncoding};
+use anyhow::{anyhow, Result};
+use clap::Args;
+use rain_orderbook_app_settings::Config;
+use rain_orderbook_common::dotrain_order::DotrainOrder;
+use std::fs::read_to_string;
+use std::path::PathBuf;
+
+#[derive(Args, Clone)]
+pub struct Orderbook {
```
Comment (hardyjosh) [inline crates/cli/src/commands/order/orderbook.rs:11]: this name is generic and confusing - should be OrderbookAddress

File: crates/cli/src/commands/order/mod.rs [hunk 3]
```diff
@@ -1,14 +1,19 @@
 mod add;
+mod calldata;
 mod compose;
 mod detail;
 mod list;
+mod orderbook;
 mod remove;
 
+use crate::commands::order::orderbook::Orderbook;
```
Comment (hardyjosh) [inline crates/cli/src/commands/order/mod.rs:9]: mod should be orderbook_address

File: crates/cli/src/commands/order/orderbook.rs (hunk 2 above)
Comment (Siddharth2207) [reply crates/cli/src/commands/order/orderbook.rs]: Resolved

File: crates/cli/src/commands/order/mod.rs (hunk 3 above)
Comment (Siddharth2207) [reply crates/cli/src/commands/order/mod.rs]: Resolved

**Review by hardyjosh — APPROVED** [review]

**Pull request merged**